tests/torch/requirements.txt

```diff
@@ -9,3 +9,6 @@ pytest-dependency>=0.5.1
 virtualenv
 # Ticket 69520
 pyparsing<3.0
+
+# Required for search_buidling_blocks tests
+transformers
```

Comment:
@vshampor hope this is ok to add such dependency for tests only.
It should be also helpful for validation JPQD feature. @vuiseng9,

Reply:
Fine by me.